civis_jupyter_notebooks/assets/initialize-git

```diff
@@ -9,13 +9,26 @@ if test $GIT_CREDENTIAL; then
   git config --global user.email "$USER_EMAIL"
   git config --global core.editor "nano"
 
+  if [[ $GIT_HOST = "github.com" ]]; then
+    USERNAME=$GIT_CREDENTIAL
+    PASSWORD=x-oauth-basic
+  elif [[ $GIT_HOST = "bitbucket.org" ]]; then
+    DECODED_CLIENT_SECRET=$(echo $BITBUCKET_CLIENT_SECRET | openssl enc -d -base64)
```

Comment:
Why do we have to decode the client secret like this? It looks like replicator is [passed the encoded credential](https://github.com/civisanalytics/console/blob/master/lib/console/services/service_manager.rb#L151) and [uses it without decoding](https://github.com/civisanalytics/replicator/blob/master/git-clone.sh#L12). Same goes for `GIT_CREDENTIAL`, which is [encoded here](https://github.com/civisanalytics/console/blob/master/lib/console/notebooks/notebook_manager.rb#L91) and used without decoding on L13 above.

Reply:
Discussed offline that we should be using k8's secretKeyRef which handles the base64 decoding - https://kubernetes.io/docs/concepts/configuration/secret/